Makefile

```diff
@@ -47,7 +55,14 @@ run-etcd:
 	--advertise-client-urls "http://$(LOCAL_IP_ENV):2379,http://127.0.0.1:2379,http://$(LOCAL_IP_ENV):4001,http://127.0.0.1:4001" \
 	--listen-client-urls "http://0.0.0.0:2379,http://0.0.0.0:4001"
 
-run-kubernetes-master: stop-kubernetes-master run-etcd
+## Run consul as a container
+run-consul: run-etcd
```

Comment:
I might be missing something, but wouldn't it make more sense for run-kubernetes-master to depend on both run-consul and run-etcd than through this intermediate step?  Unless consul somehow depends on etcd?